interview-questions.md

```diff
@@ -7,44 +7,82 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
You should give this section a bit more effort. These interview questions are your opportunity to practice what you want to say to a potential employer.